spec/requests/attributes_controller_spec.rb

```diff
@@ -3,10 +3,10 @@
 
   let(:headers) { { "Content-Type" => "application/json", "GOVUK-Account-Session" => placeholder_govuk_account_session } }
 
-  let(:attribute_name1) { "name" }
+  let(:attribute_name1) { "foo" }
   let(:attribute_value1) { { "some" => "complex", "value" => 42 } }
 
-  let(:attribute_name2) { "name2" }
+  let(:attribute_name2) { "bar" }
```

Comment:
Can we make it clearer that `attribute_name1` and `attribute_name2` need to correspond to the keys in `spec/fixtures/user_attributes.yml` for the tests to pass?

Usually, `foo` and `bar` are used as placeholders when the value is not important. In this case, that isn't entirely true: the value itself isn't meaningful but needs to match another value.

Reply:
Something like `fixtures_attribute1` and `fixtures_attribute2` perhaps?

Reply:
`foo` and `bar` are also used in the pact tests in gds-api-adapters: https://github.com/alphagov/gds-api-adapters/blob/master/test/account_api_test.rb

They weren't meaningful but, with this config file, they now are.

Reply:
If the name is meaningful, why don't we use something else that makes that clear?

Reply:
I've pushed another commit which will let gds-api-adapters be updated: 4c1c4f716d4c1bf08b569cf7d7d1df3b7f545224